### PR TITLE
Changes for publishing on thoth-station.ninja

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [x] No

### CODE REVIEW

**Review by harshad16 — COMMENTED** [review]
/lgtm 

Thanks for the changes, there is some pre-commit issues.PTAL

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            # dirty, remove trailing and leading chars
+            return line.split(" = ")[1][1:-2]
+    raise ValueError("No version identifier found")
+
+
+VERSION = get_version()
+setup(
+    name="thoth-investigator",
+    version=VERSION,
+    description="Thoth component used for manipulating with messages and workloads.",
```
Comment (goern) [inline setup.py:24]: from the readme: "Thoth's investigator is a Kafka based component that consumes all messages produced by Thoth components." ?!

Comment (fridex) [reply setup.py]: Adjusted in 45b566f, thanks!

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+"""Setup configuration for investigator."""
+
+import os
+from pathlib import Path
+from setuptools import setup
+
+
+def get_version():
+    """Get current version of adviser module."""
```
Comment (pacospace) [inline setup.py:9]: `investigator`

**Review by pacospace — APPROVED** [review]
lgtm! 💯

**Pull request merged**